packages/flutter/test/material/chip_test.dart

```diff
@@ -3497,28 +3497,28 @@ void main() {
   });
 
   testWidgets('Chip delete button tooltip can be disabled using useDeleteButtonTooltip', (WidgetTester tester) async {
+    final UniqueKey deleteButtonKey = UniqueKey();
     await tester.pumpWidget(
       _chipWithOptionalDeleteButton(
+        deleteButtonKey: deleteButtonKey,
         deletable: true,
         useDeleteButtonTooltip: false,
       ),
     );
 
-    // Tap at the delete icon of the chip, which is at the right
-    // side of the chip
-    final Offset topRightOfInkwell = tester.getTopLeft(find.byType(InkWell).first);
-    final Offset tapLocationOfDeleteButton = topRightOfInkwell + const Offset(8, 8);
-    final TestGesture tapGesture = await tester.startGesture(tapLocationOfDeleteButton);
+    // Hover over the delete icon of the chip
```

Comment:
Why do you want to change this test?

Reply:
The following tests follow almost similar procedure to test the same functionality, so this change is to make all tests have similar code.

Reply:
This is reducing our test coverage, though. The chip needs to support both modes of interaction: pressing and holding as well as hovering - and we should check for both.

Reply:
Okay, I'll revert the change.